pyproject.toml

```diff
@@ -34,7 +34,7 @@ follow_imports = "silent"
 
 [tool.poetry]
 name = "spark-client"
-version = "0.0.6"
+version = "0.0.8"
```

Comment:
we have a bit of a mess with versioning of packages. We used to have a release branch that was tagged in order to release a new python package. It is fine to upgrade this version here, but please do not tag this commit, but let's use the release branch for this.